Dreamscapes/Ldap/Core/Ldap.php

```diff
@@ -301,10 +301,12 @@ public function add($dn, array $entry)
         return $this;
     }
 
-    /** Alias of self::unbind() */
+    /**
+     * Alias of self::unbind()
+     */
     public function close()
     {
-        return $this->unbind();
+        $this->unbind();
```

Comment:
The `Ldap::close()` method is an alias for `Ldap::unbind()`. The implementation is done in the `unbind()` method. If I for whatever reason decide to change the return value of this method, I want the alias to reflect that change. `Ldap::unbind()` currently returns `void`, so this method returns also `void`.

Please revert this change.